src/sign.c

```diff
@@ -709,19 +705,17 @@ buf_findsigntype_id(buf_T *buf, // buffer whose sign we are searching for
 int
 buf_signcount(buf_T *buf, linenr_T lnum)
 {
-    sign_entry_T *sign; // a sign in the signlist
     int count = 0;
-
+    sign_entry_T *sign = NULL; // a sign in the signlist
     FOR_ALL_SIGNS_IN_BUF(buf, sign)
     {
         if (sign->se_lnum > lnum)
             // Signs are sorted by line number in the buffer. No need to check
             // for signs after the specified line number 'lnum'.
             break;
```

Comment:
```suggestion
        if (sign->se_lnum > lnum)
        {
            // Signs are sorted by line number in the buffer. No need to check
            // for signs after the specified line number 'lnum'.
            break;
        }
```

Reply:
I was wondering if it is better for readability if we put comments like that over the if instead of inside:

```c
// Signs are sorted by line number in the buffer. No need to check
// for signs after the specified line number 'lnum'.
if (sign->se_lnum > lnum)
    break;
```

Reply:
Ah, that might be better.